### ISSUE TITLE
question

### ISSUE THREAD

**Issue by iwanttoeatyo** [Danko-Lab/BayesPrism#69]
Hello,
I was wondering if non-tumors can use your software?

Thanks a lot for your help

**Comment by tinyi** [comment]
yes. simply follow the vignette and set key=NULL in run.prism function.
 examples include the PBMC shown in the BayesPrism paper.

**Comment by Lily159753** [comment]
Dear Tinyi Chu:

Hello!

My input of the Bulk RNA-seq is the combination of 3 batches, so I perform the cambat from SVM  R packages to romve the batch effect. However, despite the original data are all counts, there are negative numbers after removing batches. 

To better perform the BayesPrism, can you give me some advice on how to overcome or choose  other methods for my bulk RNA-seq preprocessing?

Thank you so much!
